src/data/docgen/react-docgen.json

```
@@ -904,6 +904,46 @@
       }
     }
   },
+  "ContainedList": {
+    "description": "",
+    "displayName": "ContainedList",
+    "props": {
+      "kind": {
+        "defaultValue": {
+          "value": "'disclosed' ? 'disclosed' : 'on-page'",
```

not sure if this is auto-generated, but should the default value just be 'on-page'?

```suggestion
          "value": "'on-page'",
```